docs/reference/semaphore-changelog.md

```diff
@@ -8,15 +8,35 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of August 31, 2020
+**(Improved) New Semaphore UI public beta release**  
+After successful private beta the new Semaphore UI has been rolled out to a larger number of users.  
+The new UI brings several improvements:  
+
+- Easier navigation through better breadcrumbs and new "Jump to" element.
+- Additional information in workflow lists allows you to quickly see the status of deployments.
+- Enhanced Workflow builder - new options like job prioritization have been added in Workflow builder.
```

Comment:
=> Enhanced Workflow builder - new options like job prioritization have been added to the Workflow builder.